src/Point/Services/Extensions/ReportedPointPositionProtoConversion.cs

```diff
@@ -0,0 +1,47 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using EulynxLive.Messages.Baseline4R1;
+using EulynxLive.Point.Proto;
+using ReportedPointPosition = EulynxLive.Messages.Baseline4R1.PointPointPositionMessageReportedPointPosition;
+
+namespace Point.Services.Extensions
+{
+    public static class ReportedPointPositionProtoConversion
+    {
+        public static PointPosition ConvertToProtoMessage(this ReportedPointPosition reportedPointPosition)
+        {
+            switch (reportedPointPosition)
```

Comment:
There's a new syntax for switch expressions that could make this even more concise

https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/operators/switch-expression